mev_inspect/classifiers/specs/compound.py

```diff
@@ -1,16 +1,76 @@
+from typing import List, Optional
+
 from mev_inspect.schemas.classifiers import (
+    Classification,
+    ClassifiedTrace,
     ClassifierSpec,
+    DecodedCallTrace,
     LiquidationClassifier,
     SeizeClassifier,
 )
+from mev_inspect.schemas.liquidations import Liquidation
 from mev_inspect.schemas.traces import Protocol
+from mev_inspect.schemas.transfers import Transfer
+
+
+class CompoundLiquidationClassifier(LiquidationClassifier):
+    @staticmethod
+    def parse_liquidation(
+        liquidation_trace: DecodedCallTrace,
+        child_transfers: List[Transfer],
+        child_traces: List[ClassifiedTrace],
+    ) -> Optional[Liquidation]:
+
+        liquidator = liquidation_trace.from_address
+        liquidated = liquidation_trace.inputs["borrower"]
+        debt_token_address = liquidation_trace.to_address
+        debt_purchase_amount = liquidation_trace.value
+        received_token_address = liquidation_trace.inputs["cTokenCollateral"]
+        received_amount = 0
+
+        debt_transfer = _get_debt_transfer(liquidator, child_transfers)
+
+        received_transfer = _get_received_transfer(liquidator, child_transfers)
+
+        seize_trace = _get_seize_call(child_traces)
+
+        if debt_transfer is not None:
+            debt_token_address = debt_transfer.token_address
+            debt_purchase_amount = debt_transfer.amount
```

Comment:
a rational for the above explicitness

if debt_transfer is none then we'll use value even if it's not cEther which could be totally wrong

is there an inputs default to be used here instead?

Reply:
yes, repayAmount, will do and separate cToken from cEther